instrumentation/opentelemetry_oban/lib/opentelemetry_oban.ex

```diff
@@ -130,15 +129,18 @@ defmodule OpentelemetryOban do
     worker = Changeset.get_field(changeset, :worker)
 
     %{
-      Trace.messaging_system() => :oban,
-      Trace.messaging_destination() => queue,
-      Trace.messaging_destination_kind() => :queue,
+      MessagingAttributes.messaging_system() => :oban,
+      MessagingAttributes.messaging_client_id() => worker,
+      MessagingAttributes.messaging_destination_name() => queue,
```

Comment:
I see here that you're keeping the existing interpretation that a "Destination" is an Oban queue.

For what is worth and for the record, noting that this other pull request (https://github.com/open-telemetry/opentelemetry-erlang-contrib/pull/436#discussion_r2499351968) is making a different interpretation in that a "Destination" is the Oban Worker and the "Consumer group" is the Oban queue.